packages/forklift-console-plugin/src/modules/Providers/views/list/ProvidersListPage.tsx

```diff
@@ -220,9 +225,9 @@ const ProvidersListPage: React.FC<{
       title={t('Providers')}
       userSettings={userSettings}
       alerts={
-        !inventoryLoading && inventoryError
```

Comment:
why do we need this changes ?

Reply:
Please see my following answer to next comment.